### DISCUSSION TITLE
关于PAGFont.RegisterFont的疑问

### DISCUSSION THREAD

**Discussion by 1456397920** [Tencent/libpag#2570] [Q&A] [answered]
Android中调用PAGFont.RegisterFont后不调用PAGFont.UnregisterFont会不会有问题，比如调用PAGFont.RegisterFont注册100个字体而不unregister它们，内存这方面会不会有问题

**Comment by kevingpqi123** [comment, marked as answer]
如果不再使用需要调用 PAGFont.UnregisterFont， PAGFont.RegisterFont 后内部会有一个 map 缓存 TypeFace， 这个 map 是全局的，如果确认后续一段时间不再使用，建议调用 PAGFont.UnregisterFont 及时清除这部分内存
